androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/splash/SplashScreen.kt

```diff
@@ -1,78 +1,244 @@
-        Box(modifier = modifier.background(gradient))
-
-        AnimatedVisibility(
-            visible = shouldShowLogo,
-            enter = fadeIn(animationSpec = tween(DURATION_LOGO))
+        Box(
+            contentAlignment = Alignment.Center,
+            modifier = modifier.fillMaxSize()
         ) {
             Image(
-                painter = painterResource(id = R.drawable.logo_white),
+                painter = painterResource(id = R.drawable.bg_splash),
                 contentDescription = null,
-                contentScale = ContentScale.Crop
+                contentScale = ContentScale.Crop,
+                modifier = Modifier
+                    .matchParentSize()
+                    .blur(animateBlur.dp)
             )
+
+            val gradient = Brush.verticalGradient(
+                colors = listOf(
+                    Black.copy(alpha = animateAlpha * InitialTopGradientAlpha),
+                    Black.copy(
+                        alpha = (animateAlpha * BottomGradientAlphaMultiplier) +
+                            InitialBottomGradientAlpha
+                    )
+                )
+            )
+            Box(
+                modifier = Modifier
+                    .matchParentSize()
+                    .background(gradient)
+            )
+
+            AnimatedVisibility(
+                visible = shouldShowLogo,
+                enter = fadeIn(animationSpec = tween(LogoDurationInMillis))
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.logo_white),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .offset(animateLogoOffset.x.dp, animateLogoOffset.y.dp)
+                        .scale(animateLogoScale)
+                )
+            }
         }
     }
 }
 
+@Composable
+private fun LoginForm(
+    onLogInClick: (email: String, password: String) -> Unit,
+    onForgotClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
+    Column(
+        verticalArrangement = Arrangement.spacedBy(20.dp),
+        modifier = modifier
+            .fillMaxSize()
+            .wrapContentHeight()
+            .padding(horizontal = 24.dp)
+    ) {
+        PrimaryTextField(
+            value = email,
+            onValueChange = { email = it },
+            placeholder = stringResource(id = R.string.login_email),
+            keyboardType = KeyboardType.Email,
+        )
+        PrimaryTextField(
+            value = password,
+            onValueChange = { password = it },
+            placeholder = stringResource(id = R.string.login_password),
+            visualTransformation = PasswordVisualTransformation(),
+            imeAction = ImeAction.Done,
+            trailingIcon = {
+                Text(
```

Comment:
Optional: I am not a fan of `.clickable` unless we cannot avoid using it. Can we try `IconButton` or `TextButton`? If it doesn't work the way we want, we can keep using `clickable`. 👍

Reply:
@kaungkhantsoe they don't have these attributes. Maybe that would make the implementation more complicated to achieve the same thing? 🤔 
![image](https://user-images.githubusercontent.com/8093908/235839019-c28c31b2-f390-4aa4-8c35-ce84278273e1.png)

Reply:
It is okay. I just want to use the widgets given by the Compose 😄 . It is optional.